### PR TITLE
Add card reader print receipt branding logo

### LINKED ISSUE #3559: Terminal card reader print receipt branding
Building upon the print receipt template created in #3224, we need to add:

* A “Powered by WooCommerce” message
* Merchant's logo

### PR DESCRIPTION
**Pull request by mordeth**
Fixes #3559

#### Changes proposed in this Pull Request

- Adding merchant branding logo on the top of the receipt.
- Adding a “Powered by WooCommerce” message on the bottom of the receipt.

Screenshot 2022-02-11 at 18 03 16

#### Testing instructions
- Update the `_wcpay_feature_card_readers` option to `1`.
- Update the `card_present_eligible` to `true`.
- Update `isBrandingEnabled` feature flag to `true` in `client/card-readers/settings/index.js`.
- Navigate to `Payments` -> `Card Readers page` -> `Receipt details`.
- Scroll down to `Branding` and upload image file to `Logo` field.
- Save settings.
- Create an order and pay it with card reader following testing on [IPP Styled Receipts: Print receipt](https://github.com/Automattic/woocommerce-payments/pull/3433)
- Using Postman make a GET request to `/wp-json/wc/v3/payments/readers/receipts/<payment_id>`.
- Verify the Logo on top and “Powered by WooCommerce” message on bottom are displayed.
-------------------

- [ ] Added changelog entries to both `readme.txt` and `changelog.txt` (or does not apply)
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

File: includes/in-person-payments/templates/html-in-person-payment-receipt.php [hunk 1]
```diff
@@ -93,11 +93,24 @@ function format_price_helper( array $product, string $currency ): string {
...
+			font-size: 7px;
+			padding-top: 5px;
+		}
+
 	</style>
 </head>
 <body>
 	<div class="receipt">
 		<div class="receipt__header">
+			<?php if ( isset( $branding_logo['file_content'] ) && isset( $branding_logo['content_type'] ) && '' !== $branding_logo['file_content'] && '' !== $branding_logo['content_type'] ) { ?>
```
Comment (anu-rock) [inline includes/in-person-payments/templates/html-in-person-payment-receipt.php:111]: Perhaps [`empty`](https://www.php.net/empty) can be used here to shorten the conditions?

File: includes/in-person-payments/templates/html-in-person-payment-receipt.php [hunk 2]
```diff
@@ -183,6 +196,7 @@ function format_price_helper( array $product, string $currency ): string {
 			<p id="application-preferred-name"><?php echo sprintf( '%s: %s', esc_html__( 'Application name', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['application_preferred_name'] ) ) ); ?></p>
 			<p id="dedicated-file-name"><?php echo sprintf( '%s: %s', esc_html__( 'AID', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['dedicated_file_name'] ) ) ); ?></p>
 			<p id="account_type"><?php echo sprintf( '%s: %s', esc_html__( 'Account Type', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['account_type'] ) ) ); ?></p>
+			<p id="credentials"><?php echo esc_html__( 'Powered by WooCommerce', 'woocommerce-payments' ); ?></p>
```
Comment (anu-rock) [inline includes/in-person-payments/templates/html-in-person-payment-receipt.php:199]: I wonder if "credentials" is the right word here. How about simply "powered_by"?

**Review by anu-rock — APPROVED** [review]
LGTM ✅ 

I have left a couple of minor comments.

File: includes/in-person-payments/templates/html-in-person-payment-receipt.php [hunk 3]
```diff
@@ -183,6 +196,7 @@ function format_price_helper( array $product, string $currency ): string {
 			<p id="application-preferred-name"><?php echo sprintf( '%s: %s', esc_html__( 'Application name', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['application_preferred_name'] ) ) ); ?></p>
 			<p id="dedicated-file-name"><?php echo sprintf( '%s: %s', esc_html__( 'AID', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['dedicated_file_name'] ) ) ); ?></p>
 			<p id="account_type"><?php echo sprintf( '%s: %s', esc_html__( 'Account Type', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['account_type'] ) ) ); ?></p>
```
Comment (mgascam) [inline includes/in-person-payments/templates/html-in-person-payment-receipt.php:198]: I know you did not touch this :) but would you mind changing this id to account-type so it follows the same pattern as the rest of id's. Thanks
```suggestion
			<p id="account-type"><?php echo sprintf( '%s: %s', esc_html__( 'Account Type', 'woocommerce-payments' ), esc_html( ucfirst( $receipt['account_type'] ) ) ); ?></p>
```

**Review by mgascam — APPROVED** [review]
LGTM! I tested the endpoint using Postman to generate the receipt and works as expected. Nice job :)

**Review by kalessil — COMMENTED** [review]
LGTM!

**Pull request merged**